src/inboxsdk.d.ts

```diff
@@ -321,16 +321,40 @@ export class RouteView extends EventEmitter {
 export interface RowDescriptor {
   /** First textual column */
   title: string;
-  /** Second textual column */
+  /**
+   * @deprecated alias for {@link RowDescriptor#snippet}.
+   */
   body: string;
+  /** Second textual column. After {@link RowDescriptor#labels} if they're provided. */
+  snippet?: string | ((args: { el: HTMLElement; onDestroy(): void }) => void);
+  /**
+   * Render an HTMLElement in the attachment icon area. This is often used to render an icon for the attachment type.
+   *
+   * @TODO do we need some sort of destroy hook here?
+   */
+  attachmentIcon?(args: { el: HTMLElement; onDestroy(): void }): void;
   /** Last text right-aligned. Often used for dates. */
   shortDetailText: string;
   /**
    * Whether the row should be rendered as read or unread similar to Gmail styles.
+   * Defaults to unread which is 700 weight text and, in a light Gmail theme, a white background.
    *
    * @TODO is this actually required like the docs say?
    */
-  isRead?: string;
+  isRead?:
+    | string
```

Comment:
```suggestion
    | boolean
```

(This should've been this way originally; this never made sense as string. It's a breaking type change but I think we should be comfortable with minor type-only breaks like this for fixes of things that were very wrong.)